client/scss/components/forms/_input-text.scss

```diff
@@ -44,6 +44,6 @@ textarea {
 .w-field--date_time_field,
 .w-field--time_field {
   input {
-    width: auto;
+    width: 100%;
```

Comment:
This forces all date/time fields to _always_ take up the full width of the parent, which is not what we want:

<img width="1068" alt="image" src="https://github.com/user-attachments/assets/d69e7de6-71a8-4338-85ef-bab9769120d3">


The issue's description specifically says **adding** `max-width`.

Have you tested your changes? Including a screenshot in your PR would be helpful for styling fixes like this one.